code/modules/projectiles/projectile/beams.dm

```diff
@@ -50,7 +50,7 @@
 /obj/item/projectile/beam/scatter
 	name = "laser pellet"
 	icon_state = "scatterlaser"
-	damage = 5
+	damage = 12.5
```

Comment:
Again dont buff this. Its op as is

Reply:
Lily this is identical to buckshot. Scatterlaser slugs were doing 15 per beam before. This isn't a buff.

Reply:
Please dont fucken buff the scatter shot gun, make a weaker verson of scatter shot thats 12.5 but dont buff the scatter shot gun.

Reply:
Scattershot laser is admin only?

Reply:
Its in a few loot tables as well as wiz-gun spawns. Ive its one of the higher end spawns.

Reply:
Wiz spawns is like such a nonissue trust me.

Reply:
It really is an issue, getting this in a wiz round is flat out the same as a pulse gun